src/rt/sched/threadpool.h

```diff
@@ -21,7 +24,7 @@ namespace verona::rt
   inline void nop() {}
 
   using namespace snmalloc;
-  template<class T>
+  template<class T, class E>
```

Comment:
Is `E` going to be `Cown` here?  I think a comment, and possibly a different letter would help.

Reply:
Switch to `C`